This will cause the Clock app to open directly to the alarm view.
Index: /usr/palm/applications/com.palm.app.clock/app/controllers/app-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.clock/app/controllers/app-assistant.js
+++ /usr/palm/applications/com.palm.app.clock/app/controllers/app-assistant.js
@@ -30,11 +30,11 @@ AppAssistant.prototype.handleLaunch = fu
 				return;
 			default:
 				Mojo.Log.info("unknown launch parameter");
-				this.launchClock();
+				this.launchAlarmList();
 				break;
 		}
 	} else {
-		this.launchClock();
+		this.launchAlarmList();
 	}
 		
 };
